File: uva113.cpp
#include<stdio.h>
#include<math.h>
int main()
{
    double a,b,c,i;
    while((scanf("%lf",&a)!=EOF))
    {
        scanf("%lf",&b);
        c=pow(b,1/a);
        printf("%.0lf\n",c);
    }
    return 0;
}

File: UVa easy problems solved in Java/uva_113.java
//package uva;

import java.util.Scanner;

public class Main {
	public static void main(String[] args)
	{
		Scanner sc=new Scanner(System.in);
		double a, b, sum;
		
		while(sc.hasNext())
		{
		a =sc.nextDouble();  
		b =sc.nextDouble();  
		
		//sum = 
		 
		sum= Math.pow(b,1/a);
		sum = Math.round(sum);
		
		System.out.println((int)sum);
		}
		 
	}

}
